nixos/modules/services/taler/config.nix

```diff
@@ -0,0 +1,143 @@
+{
+  lib,
+  pkgs,
+  config,
+  ...
+}:
+
+let
+  this = config.services.taler;
+  settingsFormat = pkgs.formats.ini { };
+in
+
+{
+  options.services.taler.settings = lib.mkOption {
+    type = lib.types.submodule {
+      freeformType = settingsFormat.type;
+      options = {
+        # Should these be in here?
+        # Should these even be configurable?
```

Comment:
Probably not, lets just hardcode them to make things easier for us

Reply:
What do you think @Atemu ?